package.json

```diff
@@ -31,6 +32,7 @@
     "eslint-plugin-jsx-a11y": "^1.2.3",
     "eslint-plugin-react": "^5.1.1",
     "istanbul": "^0.4.3",
-    "mocha": "^2.5.3"
+    "mocha": "^2.5.3",
+    "msgpack-lite": "*"
```

Comment:
@stevenc81 what version of this package should we use? I try to avoid `*` for versions it could cause tests to fail randomly in the future 😉